チャット画面の実装

udonlockin opened the pull request:
# What
チャット画面を実装する

# Why
チャットを行う際に必要だから


![](https://i.gyazo.com/4311b1ac4005be84d411c25e6da04fa5.png)

udonlockin committed:
create pull request

udonlockin committed:
Add-chat-view

TOMARUMARU commented on app/views/layouts/application.html.haml:
```diff
@@ -0,0 +1,11 @@
+!!!
+%html
+  %head
+    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
```
ハッシュロケットではなくシンボル型を使いましょう。

udonlockin replied:
ハッシュロケットをシンボル型に修正しました

TOMARUMARU commented on app/views/layouts/mailer.html.haml:
```diff
@@ -0,0 +1,8 @@
+    :css
+      /* Email styles need to be inline */
+  %body
+    = yield
```
mailerファイルは使用しないので消して大丈夫です。

udonlockin replied:
mailer.html.haml ファイルを削除しました

TOMARUMARU commented on app/views/layouts/mailer.text.haml:
```diff
@@ -0,0 +1 @@
+= yield
```
mailerファイルは使用しないので消して大丈夫です。

udonlockin replied:
mailer.text.haml ファイルを削除しました

TOMARUMARU commented on config/routes.rb:
```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  get 'messages/index'
```
rootで指定しているため、こちらは消して大丈夫です。

udonlockin replied:
指摘された2行目の get 'messages/index' 部分を削除しました

TOMARUMARU commented on app/views/layouts/application.html.haml:
```diff
@@ -0,0 +1,11 @@
+!!!
+%html
+  %head
+    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
```
ハッシュロケットではなくシンボル型を使いましょう。

udonlockin replied:
上で指摘された点と同じ内容でしょうか...？

shingoteshima replied:
同じです！失礼いたしました。

udonlockin committed:
edit

ハッシュロケットをシンボル型に変更
routes.rbの重複した記述を削除
不必要なmailerファイル以下2つを削除
・mailer.html.haml
・mailer.text.haml

udonlockin commented:
指摘された点を修正しました！

shingoteshima commented:
レビュー依頼ありがとうございます！
LGTMです！！
お疲れさまでした！
